src/game/AI/ScriptDevAI/scripts/outland/auchindoun/shadow_labyrinth/instance_shadow_labyrinth.cpp

```diff
@@ -192,10 +190,56 @@ InstanceData* GetInstanceData_instance_shadow_labyrinth(Map* pMap)
     return new instance_shadow_labyrinth(pMap);
 }
 
+
+struct go_screaming_hall_door : public GameObjectAI
+{
+    go_screaming_hall_door(GameObject* go) : GameObjectAI(go)
+    {
+        m_doorCheckNearbyPlayersTimer = 1000;
```

Comment:
put the initialization in constructor style:
, m_doorCheckNearbyPlayersTimer(1000) etc